mypy/util.py

```diff
@@ -183,24 +188,39 @@ def correct_relative_import(cur_mod_id: str,
     return cur_mod_id + (("." + target) if target else ""), ok
 
 
+FIELDS_CACHE = {}  # type: Dict[Type[object], List[str]]
```

Comment:
Style nit: Maybe use lower case name since this the dictionary object gets mutated?